docs/examples/robust_phase_estimation.ipynb

```diff
@@ -661,12 +661,12 @@
     }
    ],
    "source": [
-    "from pyquil.device import gates_in_isa\n",
-    "from pyquil.noise import decoherence_noise_with_asymmetric_ro, _decoherence_noise_model\n",
-    "# noise_model = decoherence_noise_with_asymmetric_ro(gates=gates_in_isa(qc.device.get_isa()), p00=0.92, p11=.87)\n",
+    "from pyquil.noise import decoherence_noise_with_asymmetric_ro, _decoherence_noise_model, _get_qvm_noise_supported_gates\n",
```

Comment:
Are those internal methods something that should be expose properly from pyQuil?

Reply:
🤷 This is the only place I've seen these used, so I don't know if we should move them over yet.

Reply:
Agree- a follow-up ticket after collecting all of the additional entry points sounds like the right approach.